io-engine/src/grpc/v1/stats.rs

```diff
@@ -114,18 +146,29 @@ impl StatsRpc for StatsService {
                 if let Some(name) = args.name {
                     if let Some(l) = Lvs::lookup(&name) {
                         let stats = l.base_bdev().stats_async().await?;
-                        let io_stat =
-                            IoStats::from(ExternalType((name.clone(), stats)));
+                        let io_stat = IoStats::from(ExternalType((
```

Comment:
stats_async() and IoStats::from(ExternalType()) used in both if and else, Can have a common function for it ?

Reply:
Done